mmpretrain/datasets/oxfordiiitpet.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List
+
+from mmengine import get_file_backend, list_from_file
+
+from mmpretrain.registry import DATASETS
+from .base_dataset import BaseDataset
+from .categories import OxfordIIITPet_CATEGORIES
+
+
+@DATASETS.register_module()
+class OxfordIIITPet(BaseDataset):
+    """The Oxford-IIIT Pets Dataset.
+
+    Support the `Oxford-IIIT Pets Dataset <https://www.robots.ox.ac.uk/~vgg/data/pets/>`_ Dataset.
+    After downloading and decompression, the dataset directory structure is as follows.
+
+    OxfordIIITPet_CATEGORIES dataset directory: ::
+
+        OxfordIIITPet_CATEGORIES (data_root)/
```

Comment:
The name here is not the same as it in example? Is it expected?
in example, `data_root='data/Oxford-IIIT_Pets`